pulpcore/app/tasks/test.py

```diff
@@ -18,3 +20,13 @@ def gooey_task(interval):
     from time import sleep
 
     sleep(interval)
+
+
+def group_task(inbetween=3, intervals=None):
```

Comment:
This is a userfacing name (in theory, because users should never see that task). Let's make it really obvious it's not harmful.
```suggestion
def dummy_group_task(inbetween=3, intervals=None):
```